### ISSUE TITLE
DOTImporter should propagate the vertexid with present label

### ISSUE THREAD

**Issue by jkirsch** [jgrapht/jgrapht#204]
Currently the vertex id is not propagated, in case there is a specified label.

The `VertexProvider` can not differentiate

For example for 
```dot
vertex0 [label= "Test"];
vertex1 [label= "Test"];
```

Adding the parsed vertex id to the attribute map, should at least expose that to the parser, see  [1]

```java
String label = attributes.get("label");
if (label == null) {
    label = id;
}
```

[1]: https://github.com/jgrapht/jgrapht/blob/master/jgrapht-ext/src/main/java/org/jgrapht/ext/DOTImporter.java#L583-L586

**Comment by jkinable** [comment]
Could you submit a bug fix (pull request) for this? Thanks!

**Comment by d-michail** [comment]
@jkinable After a bit of testing, I also confirm this issue. The submitted pull request fixes this.

**Comment by jkinable** [comment]
This issue has been addressed by @d-michail ; its fix will be available in jgrapht 1.0.0
